apisix/discovery/eureka.lua

```diff
@@ -66,7 +66,8 @@ local _M = {
 
 
 local function service_info()
-    local host = local_conf.eureka and local_conf.eureka.host
+    local host = local_conf.discovery and
+        local_conf.discovery.eureka and local_conf.discovery.eureka.host
```

Comment:
Could we support different instances for a same type discovery?

Reply:
This scenario is unreasonable and will not be supported for now. 
However, users can still handle this scenario by creating new instances of different gateways, I think it is enough.